source/index.md.erb

```diff
@@ -115,12 +115,6 @@ Our Zapier integration helps you integrating with the Resource Management by Sma
 * [Resource Management by Smartsheet on Zapier](https://zapier.com/apps/10000ft/integrations)
 * [Getting started with Zapier](#zapier-integration)
 
-<%# # Case studies
```

Comment:
I suppose we could put back in a link to the four case studies on the 10k landing page, but eh.

Reply:
Do we know from Product if these links/case studies documentation are used?  Seems like it redirects to the RM www page for quite sometime now (probably years?)